packages/modules/imodel-browser/src/containers/iModelTiles/IModelTile.tsx

```diff
@@ -68,6 +72,7 @@ export const IModelTile = ({
             apiOverrides={thumbnailApiOverride}
           />
         }
+        className="iac-tile"
         {...(tileProps ?? {})}
       />
     </ThemeProvider>
```

Comment:
this is broken now

<img width="816" alt="image" src="https://github.com/iTwin/admin-components-react/assets/81580355/f21d3cb3-095d-4ad8-9f45-ab55f1125045">


Was like that:
![image](https://github.com/iTwin/admin-components-react/assets/81580355/206d5457-fa6e-4446-95d0-9d601d2e64c6)

Also, double theme provider and background looks bad. Need to fix it by having single and not applying background color

Reply:
Fixed, also the background color should be fixed